src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -141,37 +138,31 @@ public Mono<Either<TransactionClosureErrorEvent, TransactionEvent<TransactionClo
-                            .transactionId(tx.getTransactionId().value().toString())
-                            .totalAmount(totalAmount)
-                            .fee(fee)
-                            .timestampOperation(updateAuthorizationRequestDto.getTimestampOperation())
-                            .paymentMethod(transactionAuthorizationRequestData.getPaymentTypeCode())
-                            .additionalPaymentInformations(
-                                    Map.of(
-                                            "outcomePaymentGateway",
-                                            transactionAuthorizationCompletedData.getAuthorizationResultDto()
-                                                    .toString(),
-                                            "authorizationCode",
-                                            authRequestData.authorizationCode(),
-                                            "rrn",
-                                            ECOMMERCE_RRN,
-                                            "fee",
-                                            fee.toString(),
-                                            "timestampOperation",
-                                            // FIXME Pass the timestamp of the authorization result
-                                            // bug CHK-1410: date formatted to yyyy-MM-ddTHH:mm:ss truncating millis
-                                            OffsetDateTime.now()
-                                                    .toLocalDateTime()
-                                                    .truncatedTo(ChronoUnit.SECONDS)
-                                                    .format(DateTimeFormatter.ISO_LOCAL_DATE_TIME),
-                                            "totalAmount",
-                                            totalAmount.toString()
-                                    )
-                            );
-
+                            .transactionId(tx.getTransactionId().value());
+
+                    if (ClosePaymentRequestV2Dto.OutcomeEnum.OK.equals(closePaymentRequest.getOutcome())) {
+                        closePaymentRequest.idPSP(transactionAuthorizationRequestData.getPspId())
+                                .idBrokerPSP(transactionAuthorizationRequestData.getBrokerName())
+                                .idChannel(transactionAuthorizationRequestData.getPspChannelCode())
+                                .totalAmount(totalAmount)
+                                .fee(fee)
+                                .timestampOperation(updateAuthorizationRequestDto.getTimestampOperation())
+                                .paymentMethod(transactionAuthorizationRequestData.getPaymentTypeCode())
+                                .additionalPaymentInformations(
+                                        new AdditionalPaymentInformationsDto()
+                                                .tipoVersamento(TIPO_VERSAMENTO_CP)
```

Comment:
This is wrong :-) 
cc @Fernando-Granato @ciuffagianluca

Reply:
No problem we define the changes to be made

Reply:
it is to be deleted